### PR TITLE
docs: Add Doc review process doc

### LINKED ISSUE #40: Document simplified document review process
Doc reviews require a lot of effort and can consume a lot of time for both the review team and the PR creator.

As such, we need to find ways to make the current process a little more streamlined for certain scenarios.

### PR DESCRIPTION
**Pull request by jodh-intel**
Create a new document that outlines the document review process.

Fixes #40.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
This is all getting very meta :)

@klynnrif, @intelkevinputnam, @grahamwhaley, @egernst - ptal.

File: Documentation-Review-Process.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+## Introduction
+
+The process for reviewing documentation PRs is slightly different from
+[the general process for reviewing code PRs](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#reviewsPR-Review-Guide.md).
+This document explains the differences.
+
+## The Documentation Team
+
+The Documentation Team are responsible for carrying out a full documentation
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: are/is: The Documentation Team is responsible for carrying out a full documentation

File: Documentation-Review-Process.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+review of a PRs containing documentation changes. Such a review includes
+checking the grammar, structure, consistency, and typographical correctness of
+the documentation changes.
+
+The following GitHub team lists the user users who are members of the Documentation Team:
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: Adding "or": The following GitHub team lists the user or users who are members of the Documentation Team:

File: Documentation-Review-Process.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+[Project maintainers](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#project-maintainers)
+have the power to approve or reject PRs using the pullapprove tool employed by
+all repositories.
+
+If a Documentation Team review is required, the pullapprove configuration will
+apply and the PR will be blocked until the required number of acks are
+obtained for the documentation changes.
+
+The technical writer from the Documentation Team will then review the document
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: Lines 44-45 suggested rewrite: The technical writer from the Documentation Team will review the document changes and add comments explaining how the wording should be changed.

File: Documentation-Review-Process.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+changes and add comments explaining how wording should be changed. It is then
+up to the PR creator to apply these changes to their original PR, rebasing as
+necessary to ensure a clean `git(1)` history. Once the creator has applied the
+changes they need to re-push their branch and add a comment to the PR asking
+for a re-review. This process can continue effectively indefinitely until the
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: ... This process continues indefinitely until the...

File: Documentation-Review-Process.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+Documentation Team is happy with the final result.
+
+## Justification for different process
+
+The documentation review process is different for the code review process for
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: The documentation review process is different than the code review process for the following reasons:

File: Documentation-Review-Process.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+the following reasons:
+
+### Users should always expect accurate Documentation
+
+Just like code changes, some documentation changes have to be merged as soon
+as possible to ensure users have the most current and accurate information.
+
+But documentation changes tend to be more urgent - consider a "Priority 1"
+code issue. This will need to be fixed with all urgency. But it would also
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: Lines 63-64 suggested rewrite: ... This needs to be fixed quickly, and also documented clearly.

File: Documentation-Review-Process.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+need to be documented clearly (and quickly). If the P1 issue cannot be fixed
+quickly, the documentation needs to be changed fast to ensure users are aware
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: Lines 65-66 suggested rewrite: quickly, update the documentation as soon as possible to alert users of the issue and to provide suitable workarounds.

File: Documentation-Review-Process.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+of the issue and are provided with suitable workarounds.
+
+Turning the problem round, if documentation is *not* kept current, users will
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: Lines 68-70 suggested rewrite: If the documentation is *not* kept current, users have incorrect information and might create problems for themselves or become disaffected with the project entirely.

File: Documentation-Review-Process.md [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+have incorrect information and may end up creating problems for themselves, or
+worse become disaffected with the project.
+
+In summary, documentation PRs should not sit in the PR backlog for any longer
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: In summary, do not let documentation PRs sit in the PR backlog for any longer than necessary.

File: Documentation-Review-Process.md [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+than necessary.
+
+### Some documentation PRs can be fast-tracked
+
+If a PR makes a minor change to one or more documents, it may not justify
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: If a PR makes a minor change to one or more documents, passing to the Documentation Team for a formal review and approval might not be necessary.

File: Documentation-Review-Process.md [hunk 11]
```diff
@@ -0,0 +1,164 @@
...
+
+   The PR only modifies code blocks embedded in the document.
+
+- "Typo" fixes
+
+  The PR only fixes spelling mistakes and whitespace issues.
+
+- Formatting
+
+  The PR only modifies the layout of the existing text (for example changing
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: Lines 117-120 suggestions (missing a closing parenthesis): 

The PR only modifies the layout of the existing text. For example, you update words to render them in a ``fixed font`` or change a [Note](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes) to fit the standard formatting conventions.

File: Documentation-Review-Process.md [hunk 12]
```diff
@@ -0,0 +1,164 @@
...
+necessary:
+
+- A new document has been added by the PR.
+
+  All documents should have an initial review unless they are extremely simple.
+
+## Requesting a formal document review
+
+If a PR needs a full document review add a comment to the PR like the
+following which will send a mail to all members of the Documentation Team:
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: following, which sends a mail to all members of the Documentation Team:

File: Documentation-Review-Process.md [hunk 13]
```diff
@@ -0,0 +1,164 @@
...
+1. Request a project maintainer force-merges the PR.
+
+   This operation bypasses the pullapprove check that will stop a PR from
+   landing until the Documentation Team ack the PR.
+
+1. The maintainer should notify the Documentation Team out of courtesy that
+   the PR has been merged.
+
+   This allows a follow-on "recovery PR" to be raised should the Documentation
+   Team disagree that the lightweight process should have been used.
```
Comment (klynnrif) [inline Documentation-Review-Process.md]: Team disagree on using the lightweight process.

File: Documentation-Review-Process.md (hunk 2 above)
Comment (jodh-intel) [reply Documentation-Review-Process.md]: Sorry - my mistake. The word "user" should not have been there, so I've just dropped it to become:

> The following GitHub team lists the users who are members of the Documentation Team:

**Comment by jodh-intel** [comment]
Thanks @klynnrif - branch updated.

File: Documentation-Review-Process.md [hunk 14]
```diff
@@ -0,0 +1,164 @@
...
+
+[Project maintainers](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#project-maintainers)
+have the power to approve or reject PRs using the pullapprove tool employed by
+all repositories.
+
+If a Documentation Team review is required, the pullapprove configuration will
+apply and the PR will be blocked until the required number of acks are
+obtained for the documentation changes.
+
+The technical writer from the Documentation Team will review the document
```
Comment (grahamwhaley) [inline Documentation-Review-Process.md]: Not sure on 'The technical writer'. Maybe better as 'A technical writer', or just 'Technical writers' ?

File: Documentation-Review-Process.md [hunk 15]
```diff
@@ -0,0 +1,164 @@
...
+
+## Justification for different process
+
+The documentation review process is different than the code review process for
+the following reasons:
+
+### Users should always expect accurate Documentation
+
+Just like code changes, some documentation changes have to be merged as soon
+as possible to ensure users have the most current and accurate information.
```
Comment (grahamwhaley) [inline Documentation-Review-Process.md]: I'd drop 'the most'

File: Documentation-Review-Process.md [hunk 16]
```diff
@@ -0,0 +1,164 @@
...
+
+But documentation changes tend to be more urgent - consider a "Priority 1"
```
Comment (grahamwhaley) [inline Documentation-Review-Process.md]: starting a sentence with 'but'? :-) Also, is that s/consider/considered/ ?
I'm actually having trouble parsing this whole paragraph - can you have a re-read and see if it reads OK for me?

File: Documentation-Review-Process.md [hunk 17]
```diff
@@ -0,0 +1,164 @@
...
+
+  All documents should have an initial review unless they are extremely simple.
+
+## Requesting a formal document review
+
+If a PR needs a full document review add a comment to the PR like the
+following, which sends a mail to all members of the Documentation Team:
+
+```
+ping @kata-containers/documentation
```
Comment (grahamwhaley) [inline Documentation-Review-Process.md]: maybe s/ping/PTAL/

File: Documentation-Review-Process.md [hunk 18]
```diff
@@ -0,0 +1,164 @@
...
+1. Obtain general agreement that
+   a Document Team review [is not required](#decision-authority).
+
+1. Ensure all other approvals and CI checks have been met.
+
+1. Ensure the PR contains a note explaining why the PR does not justify a full review.
+
+1. Request a project maintainer force-merges the PR.
+
+   This operation bypasses the pullapprove check that will stop a PR from
```
Comment (grahamwhaley) [inline Documentation-Review-Process.md]: s/will stop/normally stops/

File: Documentation-Review-Process.md (hunk 14 above)
Comment (jodh-intel) [reply Documentation-Review-Process.md]: Fixed.

File: Documentation-Review-Process.md (hunk 15 above)
Comment (jodh-intel) [reply Documentation-Review-Process.md]: Fixed.

File: Documentation-Review-Process.md (hunk 16 above)
Comment (jodh-intel) [reply Documentation-Review-Process.md]: Reworded for clarity.

File: Documentation-Review-Process.md (hunk 17 above)
Comment (jodh-intel) [reply Documentation-Review-Process.md]: Fixed.

File: Documentation-Review-Process.md (hunk 18 above)
Comment (jodh-intel) [reply Documentation-Review-Process.md]: Fixed.

File: Documentation-Review-Process.md [hunk 19]
```diff
@@ -0,0 +1,164 @@
...
+
+The documentation review process is different than the code review process for
+the following reasons:
+
+### Users should always expect accurate Documentation
+
+Just like code changes, some documentation changes have to be merged as soon
+as possible to ensure users have current and accurate information.
+
+Documentation changes tend to be more urgent though. For example a "Priority
```
Comment (grahamwhaley) [inline Documentation-Review-Process.md]: nitpick: s/For example a/For example, a/

**Comment by grahamwhaley** [comment]
Looks like you still need another @kata-containers/community ack though...

**Comment by jodh-intel** [comment]
One more approval needed over here so we can get this landed...

**Comment by jodh-intel** [comment]
Ping @kata-containers/community - we need this landed folks!

**Pull request merged**